### DISCUSSION TITLE
Give crawler mask and cultist robes clown tags to trigger clown power

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12732] [Ideas and suggestions]
Clown power can often provide many benefits but for people role playing as members of the church they must deny themselves of all of the benefits. Doing this will allow them to use most of them barring the horn and hammer ones while also doing nothing to alter gameplay as both items provide no stats themselves. If it's a concern have clown power invert disguised as husk.

**Comment by Next-Door-Tech** [comment]
In universe the Church and the Children are at odds, so definite -1 on the cultist robes here. Allowing the benefits of both clown power and the cultist clothing would also be a bit OP IMO - those benefits should stay mutually exclusive.

**Reply by Hazmat616** [reply]
The only benefit the cultist robe provides is hiding from husks. I'm not talking about zealot robes.
